test-podinfo/zarf.yaml

```diff
@@ -0,0 +1,25 @@
+kind: ZarfPackageConfig
+metadata:
+  name: test-podinfo
+  description: Example showcasing multiple ways to deploy helm charts
+  version: 0.1.0
+
+components:
+  - name: demo-helm-charts
+    required: true
+    charts:
+      - name: uds-podinfo-config
+        namespace: podinfo
+        version: 0.1.0
+        localPath: config-chart
+      - name: podinfo-local
```

Comment:
Is it possible to use the OCI published version as opposed to pulling in the whole chart?